recipes/code-server/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -euo pipefail
+set -x
+
+VERSION=${PKG_VERSION//_/-}
+
+vscodeVersion=$(python -c "print('${VERSION}'.split('-vsc')[1])")
+codeServerVersion=$(python -c "print('${VERSION}'.split('-vsc')[0])")
+
+yarn
+MINIFY=true yarn build "${vscodeVersion}" "${codeServerVersion}"
+yarn binary "${vscodeVersion}" "${codeServerVersion}"
+mkdir -p ${PREFIX}/bin/
+mv binaries/code-server${VERSION}-*-x86_64 ${PREFIX}/bin/
```

Comment:
Maybe it should be `mv binaries/code-server${VERSION}-*-x86_64 ${PREFIX}/bin/code-server` instead, so the executable always has the name c`ode-server`, which is required for the tests.

Reply:
Yes. I'm sadly to test this locally as the recipe won't build for me on OSX due to https://github.com/conda-forge/git-feedstock/issues/50 (which is not happening in CI). I therefore run the steps without conda-build and copied the necessary commands over here. The last two lines are the untested ones :(